internal/consensus/reactor.go

```diff
@@ -923,47 +934,59 @@ func pickVoteCurrentHeight(
 	// If there are lastCommits to send...
 	if prs.Step == cstypes.RoundStepNewHeight {
 		if vote := ps.PickVoteToSend(rs.LastCommit); vote != nil {
-			logger.Debug("Picked rs.LastCommit to send")
+			if logger.DebugOn() {
+				logger.Debug("Picked rs.LastCommit to send")
```

Comment:
So, this kind of debug log does not constitute a problem. The string is a constant, included in the binary (whether debug is enabled or not). And the cost of checking the level can be paid inside the `Debug()` call.